### DISCUSSION TITLE
Researcher - Federated search to get data from the performance/theatre domain

### DISCUSSION THREAD

**Discussion by KlausIllmayer** [pair-cg/general-contributions#51] [User Stories]
# 1. Need:
I like to have a single research website where I can look for data entities from the performance/theatre domain on as many as possible databases.
If I search e.g. for a work (as an example a drama) I like to see where this work was performed across countries and times. I could also look for an actor/actress to see all the performances where s/he took part.

# 2. Action:
Set up a single website as a starting point to search for theatre/performance data across different and as many as possible databases (= federated search).
Give a good documentation at hand how to best use this website and the search queries for my research.

# 3. Desired Outcome:
A search result that gives me all available data on the data entity that I'm looking for. In a more sophisticated version it could give me also tailored visualisations dependent from the type of search data (like movements between places for data type persons or frequency of performances per year fro data type performances).

# 4. Perspective:
Researcher

# 5. Stakeholder Group:
Data providers
Researchers

# 6. Additional info:
Trying to explain this use case from a more technical perspective in a dedicated use case.

**Comment by drosu** [comment]
Hi Klaus. 
(1) Are the following two formulations  close to the needs you had in mind?
     (a) "I want to be able to assess the global popularity of a work (e.g., a drama) based on when and where in the world this work was performed since its publication." 
    ( b) " I want to asses the professional activity of actor/actress based on all the performances they took part in since  they started their career."
The context you described also highlight issues related to publishing information about the performance of works in translation (e.g., the need to also share information about the original work). For example, Arthur Schnitzler's play Der Reigen has been staged several times in Romania, under different names, e.g., "Cercul" at Teatrul Mic (Bucharest, 1995) and "Hora Iubirilor" at Teatrul Nottara (Bucharest, 2002), but the connection between these stagings and Der Reigen can only be made using "third-party" information, e.g., online CVs of the actors who participated in the stagings, Arts and Culture articles in the Romanian press, etc.
(2) A central database that collects makes available information about performances worldwide  is a potential technological solution for fulfilling the needs above, rather than a need itself.

**Reply by KlausIllmayer** [reply]
Yes, (1a) and (1b) can be seen as iterations of this user story (depending on the research question).
Regarding different notations of the title of a work: this is something that is connected to the way the common data model/ontology handles this and how far authority file identifiers are used. The user role "researcher" likes to be unaware abouth how this is handled. But in general, it is clear that the work "Der Reigen" has a graduated relation to a performance that is based on this work (it could be a 1:1 relation but usually for a performance there are deletions or even only allusions to a work - it depends on the concept, the aesthetics, etc., you can never be sure how much of the work is adopted in the performance). The type of relation between work and performance could express the type of adaption to a work (but as you already pointed out: this can only be said after doing a research and consulting archives etc.). I guess we can fill books with discussions on this. It would be great if the PAIR CG could give recommendations on how to model different relations between work and performance.

Regarding (2): A federated search do not necessarily rely on a central database. The user role "researcher" does not really care about the technological solution. What the user role "researcher" needs is a single access point (central website) to fire queries for a federated search and get results. Therefore the need for "researcher" is not the technological solution but the single access point.

**Reply by drosu** [reply]
(1) 
(A) Concrete examples of research questions are very important, and most welcome, because they will highlight the type of data/information that needs to be published / shared / collected.
(B) As for issues related to "named entities", recognizing entities, linking entities, etc., this might be outside the scope of PAIR-CG, but definitely warrants a thorough discussion. 
(C)Identifying the type of relationships between an original work and performances (e.g., adaption of a work, faithful/unabridged translation, abridged translation, censored translation, performance that references the entire work or just a character, idea, etc.) are squarely within the mission of PAIR-CG.
(2)I think a clarification might be helpful: the point I was trying to make is that technology helps fulfill needs, but it is not a need in itself.
